cohorts/functions.py

```diff
@@ -84,7 +84,8 @@ def count_filter_fn(filterable_variant, **kwargs):
             return ((filterable_variant_function(filterable_variant) if filterable_variant_function is not None else True) and
                     filter_fn(filterable_variant, **kwargs))
         patient_id = row["patient_id"]
-        return cohort.load_variants(
+        return cohort.load_effects(
```

Comment:
re: getting rid of both function builders, I will likely only get rid of one -- the function builder is still useful for some edge cases not addressed by the function builder . E.g.: `missense_snv_and_nonsynonymous_indel_count` where the logic contains an _or_ (is_indel or (is_snv & is_substitution). It would be nice to generalize the filter-fn logic further to enable a phrase like that above, but i'll leave that for another day ;)